Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM php:7.4-cli
+
+RUN apt update && apt install libgearman-dev -y && pecl install gearman
+
+RUN docker-php-ext-enable gearman \
+    && docker-php-ext-configure pcntl --enable-pcntl \
+    &&  docker-php-ext-install pcntl
+
+COPY .docker/php.ini /usr/local/etc/php/php.ini
+
+WORKDIR /app
```

Comment:
Since the Dockerfile and docker-compose.yaml are just needed for developer experience I think we should consider placing them in a subfolder or renaming them so they are not automatically picked up. We can still advise in the README how the docker container can be started. Let's discuss tomorrow.

Reply:
Is there any other way to set up docker for development? 

I don't have enough info on search. I'm happy to insert any additional info inside Readme, just feel free to let me know what needs to be added.

Reply:
Created the subfolder and added the file in there in my last commit